tests/contrib/scrapy/test_pipelines.py

```diff
@@ -210,6 +212,25 @@ class PipelineJSONSchemaValidator(PipelineTest):
     ]
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 4), reason="mock requires python3.4 or higher"
+)
+def test_validator_from_url():
+    from unittest import mock
+
+    with mock.patch("spidermon.utils.web.get_contents") as mocked_get_contents:
+        mocked_get_contents.return_value = test_schema_string
+        settings = {SETTING_SCHEMAS: {TestItem: "https://fixtures.com/testschema.json"}}
+        test_item = TestItem()
+        crawler = get_crawler(settings_dict=settings)
+        pipe = ItemValidationPipeline.from_crawler(crawler)
```

Comment:
Maybe checking the stats of the pipeline isn't really testing if the JSON validator is using the content returned by the schema you provided. I thought in two different tests that may check better this scenario:
- Verify if the `ItemValidationPipeline` returned contains inside `validators` a `JSONSchemaValidator` containing `test_schema_string` in `_schema` or;
- When creating an `ItemValidationPipeline`, verify if `JSONSchemaValidator` was created using `test_schema_string` and this validator appears in `validators`of the pipeline.